README.md

```diff
@@ -63,6 +63,44 @@ ls | at -t `python -m cats -d 5 --loc OX1`
 
 ![CATS animated usage example](cats.gif)
 
+#### Displaying carbon footprint estimates
+
+`cats` is able to provide an estimate for the carbon footprint
+reduction resulting from delaying your job.  To enable the footprint
+estimation, you must provide information about the machine in the form
+of a YAML configuration file.  An example is given below:
+
+```yaml
+cluster_name: "CW23"
+postcode: "EH8 9BT"
```

Comment:
This should now be `location` instead of postcode

Reply:
Done as I rebased this branch on `main`.